api/database/migrations/2022_06_22_170914_update_pool_candidatate_filter_employment_duration.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdatePoolCandidatateFilterEmploymentDuration extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('pool_candidate_filters', function (Blueprint $table) {
+            $table->boolean('would_accept_temporary')->nullable();
```

Comment:
While the query itself works fine as a boolean we should probably save this as an enum/string so we can actually tell whether a user selected 'any duration' or 'permanent'.

Reply:
I realized I didn't even need to add that into this PR, I didn't need any API/DB changes.
That part is stealing from #3106 without real reason so just going to undo the migration bit 😬  🤣

Reply:
Sounds good!